composer.json

```diff
@@ -36,7 +36,9 @@
         "vlucas/phpdotenv" : "2.5.1",
         "yubico/u2flib-server" : "1.0.2",
         "zendframework/zendframework" : "2.5.3",
-        "particle/validator": "2.3.3"
+        "particle/validator": "2.3.3",
+        "guzzlehttp/guzzle": "6.3.3",
+        "kamermans/guzzle-oauth2-subscriber": "^1.0.5"
```

Comment:
better to just strictly hard-code the version:
```
"kamermans/guzzle-oauth2-subscriber": "1.0.5"
```
(ie. drop the carat)

Reply:
Thought I had thus the 1.0.5. Chances are i'm sleep developing again:) will change.